apps/api/src/app/sessions/audio-session.interceptor.ts

```diff
@@ -0,0 +1,25 @@
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor } from '@nestjs/common';
+import { Observable, map } from 'rxjs';
+import { Session } from './entities/session.entity';
+import { isArray } from 'class-validator';
+
+@Injectable()
+export class AudioSessionInterceptor implements NestInterceptor {
+  intercept(context: ExecutionContext, next: CallHandler): Observable<Session | Session[]> {
+    return next.handle().pipe(
+      map((data: any) => {
+        if (isArray(data)) {
+          data.map((session: Session) => {
+            session.isAudioSession = !!session.mediaId;
```

Comment:
Do we need an interceptor here? Can't we just add this field as a getter to the session entity?

Reply:
Currently this is only used on UI level not on backend level. That's why I added it only in the layer where the session entity is requested (of course I could have added it in the resolver itself, but the interceptor felt like a better place to do something in addition for some of the requests - the subscription request should most likely be excluded in the future). Adding it in a lower layer would always calculate it and add it to the entity even for "internal" requests